src/ui/public/sortable_column/sortable_column.less

```diff
@@ -0,0 +1,9 @@
+@import (reference) "~ui/styles/variables";
+@import (reference) "~ui/styles/mixins";
+@import (reference) "~ui/styles/theme";
+
+sortable-column {
+  .sortable-column-liner {
+    display: inline-block;
+  }
```

Comment:
We should avoid nested selectors where possible, and we should avoid element selectors because they create a tight coupling with the directive. Let's use classes instead here.